### DISCUSSION TITLE
MISSING PACKAGES

### DISCUSSION THREAD

**Discussion by marticliment** [Devolutions/UniGetUI#1973] [Announcements]
### If you are running WingetUI 3.0.1 or higher:
1. Go to WingetUI Settings -> Winget Preferences and click `Reset Winget Sources`
2. Restart WingetUI

### If you are running WingetUI 3.0.0 or lower
1. Quit WingetUI
2. Delete the `C:\Users\<EVERY-USER>\AppData\Local\Microsoft\WinGet\Settings\defaultState` folder and its contents. You may be requested for administrator permission. Click "Yes". **DO THIS FOR EVERY USER ON YOUR MACHINE**
3. Start WingetUI and navigate to Settings -> Winget Preferences and click on  `Reset Winget Sources`
4. Restart WingetUI again and disable System Winget if you had enabled it.ç
5. Restart WingetUI a final time.

**Comment by Kim-Ko** [comment]
Hi, 

any assistance, advice or pointers to resources / documentation would be appreciated. 
Just created a GetHub account to seek assistance. WingetUI has ceased to function on my PC.
Have tried the uninstall, reinstall of version 3.0.2.  
I did the reset and relaunched under the Winget source in the settings. No list of installed applications showed back up.
I can do a list with Winget and see applications I have installed.
Probably just got luck this worked the first time I tried it. 

Update, thought it was still working on my laptop (Version 2.2.0, but just checked and installed packages is blank there as well

Kim
